### DISCUSSION TITLE
checkRequiredFile

### DISCUSSION THREAD

**Discussion by konys30** [dongaba/TVerRec#37] [Q&A] [answered]
I run a.download_video.cmd, and it will display
【checkRequiredFile: D:\TVerRec-2.7.2\src\tverrec_bulk.ps1:114
Line |
 114 |  checkRequiredFile
     |  ~~~~~~~~~~~~~~~~~
     | 番組ダウンロード先ディレクトリが存在しません。終了します。】
how to fix this?

**Comment by dongaba** [comment]
You first need to update your setting file to TVerRec to work properly.
You can refer to [Wiki page](https://github.com/dongaba/TVerRec/wiki) to know what needs to be set up and how you can configure TVerRec.
The Wiki is written in Japanese but you can use Google Translate or something similar to translate in your preferred language.

**Reply by konys30** [reply]
The translator tells me that  “番組ダウンロード先ディレクトリが存在しません” means the download directory is not exist.
I think the download directory should be created automatically or included in the original file.
I didn't find what I needed in the Wiki.

**Reply by dongaba** [reply, marked as answer]
番組ダウンロード先ディレクトリ is a directory where you would like to download video files and can be specified with `$script:downloadBaseDir` in the setting file.

You also need to specify `$script:downloadWorkDir` as 作業ディレクトリ which means the working directory.
You can specify 
移動先ディレクトリ which is a media library directory with `$script:saveBaseDir` but this can be left blank.

Hope this helps.

**Reply by konys30** [reply]
Thanks! 
Following your suggestion,now I can use it normally.
I checked the wiki again and found that the problem has been described in detail in the 基本的な設定.
I'm sorry to bother you for my negligence.

**Reply by dongaba** [reply]
Glad to hear that you can use TVerRec now, enjoy!
